Fixing typo in readme (#3070)
sveltejs/kit: packages/adapter-cloudflare-workers/README.md

  Cloudflare Pages

> **Note:** Cloudflare Pages' new Workers integration is currently in beta.<br/>

> > The `adapter-cloudflare`, adapter will be the preferred approach for most users since building on top of Pages unlocks automatic builds and deploys, preview deployments, instant rollbacks, etc.<br/>

Reviewer: ```suggestion
> Compared to `adapter-cloudflare-workers`, `adapter-cloudflare` is the preferred approach for most users since building on top of Pages unlocks automatic builds and deploys, preview deployments, instant rollbacks, etc.<br/>
```
